source/iface/context.rst

```diff
@@ -149,22 +153,40 @@ information.
    * - Descriptor
      - Return type
      - Description
-   * - reference_count
-     - ``cl_uint``
-     - Reference count of the underlying cl_context
    * - platform
      - :ref:`platform`
-     - SYCL platform for the context
+     - Returns the platform associated with the context.
    * - devices
-     - ``vector_class<device>``
-     - SYCL devices associated with this platform
-
+     - ``std::vector<device>``
+     - Returns all of the devices associated with the context.
+   * - atomic_memory_order_capabilities
+     - ``std::vector<memory_order>``
+     - Returns the set of memory orderings supported by atomic operations
+       on all devices in the context, which is guaranteed to include ``relaxed``.
+       The memory ordering of the context determines the behavior of atomic
+       operations applied to any memory that can be concurrently accessed by
+       multiple devices in the context.
+   * - atomic_fence_order_capabilities
+     - ``std::vector<memory_order>``
+     -  Returns the set of memory orderings supported by ``atomic_fence`` on all
+        devices in the context, which is guaranteed to include ``relaxed``,
+        ``acquire``, ``release`` and ``acq_rel``.
+        The memory ordering of the context determines the behavior of fence operations
+        applied to any memory that can be concurrently accessed by multiple devices in the context.
+   * - atomic_memory_scope_capabilities
+     - ``std::vector<memory_scope>``
+     -  Returns the set of memory scopes supported by atomic operations on all
+        devices in the context, which is guaranteed to include ``work_group``.
+   * - atomic_fence_scope_capabilities
+     - ``std::vector<memory_scope>``
+     -  Returns the set of memory orderings supported by ``atomic_fence`` on all
+        devices in the context, which is guaranteed to include ``work_group``.
```

Comment:
The behavior of all four of these "capability" queries has been clarified since the last published version of the spec.  Can you use the updated descriptions here?  You can either look at the most recent CI build of the spec, or see KhronosGroup/SYCL-Docs#382.

Reply:
The descriptions updated with : https://github.com/KhronosGroup/SYCL_Reference/pull/73/commits/912d143a253b68aa9b379715c9e5be2fff40b12f